File: codeforces/cf1521A_nastia_and_nearly_good_numbers.py
# uninhm
# https://codeforces.com/contest/1521/problem/A
# implementation

for _ in range(int(input())):
    a, b = map(int, input().split())

    if b == 1:
        print("NO")
        continue

    x = a
    y = a*b
    z = x+y

    print("YES")
    print(x, y, z)

File: codeforces/cf1521A_nastia_and_nearly_good_numbers.hs
-- user: nel_tu_
-- path: https://codeforces.com/contest/1521/problem/A
-- tags: math

solve [_, 1] = Nothing
solve [a, b] = Just [a, a*b, a*b + a]


input = map (map read . words) . tail . lines
output = maybe "NO" $ ("YES\n" ++) . unwords . map show

main = interact $ unlines . map (output . solve) . input

